db/migrations/0005_alter_movie_actor_alter_movie_genre.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 4.0.2 on 2024-11-27 00:16
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('db', '0004_alter_movie_actor_alter_movie_genre'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='movie',
+            name='actor',
+            field=models.ManyToManyField(related_name='actors', to='db.Actor'),
```

Comment:
The `related_name` for the `actor` field should be `movies` instead of `actors` to correctly represent the reverse relation from `Actor` to `Movie`.